SYCL/Regression/kernel_bundle_ignore_sycl_external.cpp

```diff
@@ -0,0 +1,45 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
```

Comment:
Does it make sense to run on the host device if the test skips execution on such a device(line 22)?

Reply:
Good point. I don't think so, so I have removed it.